### PR TITLE
Some installation performance improvements

### PR DESCRIPTION
**Pull request by dasmb**
Our performance with what I'm going to call "global clusters" -- install machine, binary repositories and target infrastructure geographically distinct from each other -- is not up to par. And while we can encourage KET users to cache binaries locally and use an install machine that is geographically close to the cluster, there are certainly some tweaks we can perform to improve SSH & download performance for these users.

Note that this PR uses an experimental CloudFront repo for binaries. This causes issues when testing a binary release as it can take an indeterminate time, up to an hour, for all changes to propagate and propagation order matters.

### CODE REVIEW

File: pkg/ansible/runner.go [hunk 1]
```diff
@@ -154,7 +157,12 @@ func (r *runner) startPlaybook(playbookFile string, inv Inventory, vars ExtraVar
 	os.Setenv("ANSIBLE_JSON_LINES_PIPE", r.namedPipe)
 
 	// Print Ansible command
-	fmt.Fprintln(r.out, strings.Join(cmd.Args, " "))
+	fmt.Fprintf(r.out, "export PYTHONPATH=%v\n", os.Getenv("PYTHONPATH"))
+	fmt.Fprintf(r.out, "export ANSIBLE_CALLBACK_PLUGINS=%v\n", os.Getenv("ANSIBLE_CALLBACK_PLUGINS"))
+	fmt.Fprintf(r.out, "export ANSIBLE_CALLBACK_WHITELIST=%v\n", os.Getenv("ANSIBLE_CALLBACK_WHITELIST"))
+	fmt.Fprintf(r.out, "export ANSIBLE_CONFIG=%v\n", os.Getenv("ANSIBLE_CONFIG"))
```
Comment (dkoshkin) [inline pkg/ansible/runner.go]: What are these doing?

**Comment by alexbrand** [comment]
LGTM!

File: pkg/ansible/runner.go (hunk 1 above)
Comment (dasmb) [inline pkg/ansible/runner.go]: We were previously outputting the ansible command line to an ansible log file, ostensibly for debug purposes. But that command is useless without these corresponding environment variables, so I've added them to the output.

File: ansible/group_vars/all.yaml [hunk 2]
```diff
@@ -166,8 +166,8 @@ pause_img: gcr.io/google_containers/pause-amd64:{{ pause_version }}
 allow_package_installation: true
 yum_repository_url: https://kismatic-packages-rpm.s3-accelerate.amazonaws.com
 yum_gpg_key_url: https://kismatic-packages-rpm.s3-accelerate.amazonaws.com/public.key
-deb_repository_url: https://kismatic-packages-deb.s3-accelerate.amazonaws.com
-deb_gpg_key_url: https://kismatic-packages-deb.s3-accelerate.amazonaws.com/public.key
+deb_repository_url: https://d38kcxml09yeoe.cloudfront.net
+deb_gpg_key_url: https://d38kcxml09yeoe.cloudfront.net/public.key
```
Comment (dkoshkin) [inline ansible/group_vars/all.yaml]: Are we switching to CF? If so why only for debs?

Comment (alexbrand) [inline ansible/group_vars/all.yaml]: Ah! I glanced over this thinking it was switching both GPG keys to CF.

**Comment by dkoshkin** [comment]
LGTM

**Comment by alexbrand** [comment]
Failure seems like a flake, so I am merging this in.

**Pull request merged**